build_tools/setup_helpers/extension.py

```diff
@@ -107,8 +109,36 @@ def _build_sentence_piece(debug):
     )
 
 
+def _build_re2(debug):
+    build_dir = _TP_BASE_DIR / 're2' / 'build_dir'
+    build_dir.mkdir(exist_ok=True)
+    build_env = os.environ.copy()
+    config = 'Debug' if debug else 'Release'
+    if platform.system() == 'Windows':
+        extra_args = ['-GNinja']
+        build_env.setdefault('CC', 'cl')
+        build_env.setdefault('CXX', 'cl')
+    else:
+        extra_args = []
+    subprocess.run(
+        args=['cmake', f'-DCMAKE_CXX_FLAGS=-fPIC', f'-DBUILD_SHARED_LIBS=OFF',
```

Comment:
BTW, could you please remove the flag `fPIC` for Windows?

Reply:
what flag should be used instead?